connectors/src/connectors/google_drive/temporal/file.ts

```diff
@@ -343,13 +346,21 @@ export async function syncOneFile(
             const tableName = slugify(file.name.substring(0, 32));
             const tableDescription = `Structured data from Google Drive (${file.name})`;
 
+            const delimiter = await guessDelimiter(tableCsv);
+            const records = parse(tableCsv, {
+              columns: (c) => {
+                return getSanitizedHeaders(c);
+              },
+              delimiter,
+            });
+            const stringifiedContent = stringify(records, { header: true });
```

Comment:
Can we move all of this in types so that each project don't need to have `csv-parse` as a direct dependency?

Reply:
Sure 😃